player.js

```diff
@@ -40,32 +40,34 @@ Player.prototype.move = function(velocity){
   var angle = this.orientation * Math.PI / 180
   this.position.x += velocity.x*Math.cos(angle)-velocity.y*Math.sin(angle);
   this.position.y += velocity.x*Math.sin(angle)+velocity.y*Math.cos(angle);
+//  this.position.x += velocity.x;
+//  this.position.y += velocity.y;
 };
 
 Player.prototype.checkBoundaries = function(){
-  if (this.position.x <= 0){
-    this.position.x = 0;
+  if (this.position.x <= -this.game.width/4 + this.size.x){
+    this.position.x = -this.game.width/4 + this.size.x;
   }
 
-  if (this.position.x >= this.game.width - this.size.x){
-    this.position.x = this.game.width - this.size.x;
+  if (this.position.x >= this.game.width/4 - this.size.x){
+    this.position.x = this.game.width/4 - this.size.x;
   }
 
-  if (this.position.y <= 0){
-    this.position.y = 0;
+  if (this.position.y <= -this.game.height/4 + this.size.y){
+    this.position.y = -this.game.height/4 + this.size.y;
   }
 
-  if (this.position.y >= this.game.height - this.size.y){
-    this.position.y = this.game.height - this.size.y;
+  if (this.position.y >= this.game.height/4 - this.size.y){
+    this.position.y = this.game.height/4 - this.size.y;
   }
 };
 
 Player.prototype.keyboardInput = function(keyboard){
-  if ('L' in keyboard.keysDown){
+  if ('O' in keyboard.keysDown){
```

Comment:
I feel like the player should always be controlled by WASD. Since that's the most common control scheme, and I imagine that we will almost exclusively use it, we should reserve the less common controls for the camera.